code/__HELPERS/ai.dm

```diff
@@ -0,0 +1,35 @@
+//Various macros
+#define GET_WEIGHT(NODE, NAME) NODE.weights[NAME]
+#define WEIGHT_NOT_ZERO(NODE, NAME) weights[NAME] != 0
```

Comment:
you dont need this, just do `weights[NAME]` in the code, if its unset this will be true since `null != 0`